config/locales/en.yml

```diff
@@ -92,11 +92,11 @@ en:
         revoke?: You are not authorized to revoke personal access tokens for user %{name} on this server.
         update?: You are not authorized to update user %{name} on this server.
       workflow_execution:
-        cancel?: You are not authorized to cancel workflow executions for %{namespace_type} %{name}
-        create?: You are not authorized to submit workflow executions for %{namespace_type} %{name}
-        destroy?: You are not authorized to destroy workflow executions for %{namespace_type} %{name}
-        index?: You are not authorized to view workflow executions for %{namespace_type} %{name}
-        read?: You are not authorized to view workflow execution %{id}
+        cancel?: You are not authorized to cancel pipeline executions for %{namespace_type} %{name}
```

Comment:
**nitpick (blocking):** The following 4 translations should be reverted to `workflow executions`